### ISSUE TITLE
crash when one of `gitlabUserName` and or `gitlabUserUsername` is undefined

### ISSUE THREAD

**Issue by ktdreyer** [red-hat-storage/rdopkg-tar#33]
This bit of code crashes when one of the environment variables is undefined:

```python
    userName = os.environ.get('gitlabUserName')
    gitlabuserName = os.environ.get('gitlabUserUsername')

    if userName or gitlabuserName:
        commit_distgit_amend(suffix='GitLab-User: ' + gitlabuserName + ' (' + userName + ')')
```
Specifically, we cannot concatenate `None` with a string:

```
TypeError: can only concatenate str (not "NoneType") to str
```

**Comment by ychai23** [comment]
I am adjusting the code to make both field required in order to log the user name is that fine? so change the or to and

**Comment by bigjust** [comment]
Fixed with #39
